flexport/latitude
# LDS-371 MultiInput allows Pill text to be highlighted, but de-highlights on blur

## Description:
Prevents Pills from being selectable when in the `MultiInput` component. Adds a new prop `selectable` to `Pill` to prevent selectability.

## Jira:
[LDS-371](https://flexport.atlassian.net/browse/LDS-371)

## Manual Testing Plan:
1. Load Storybook.
2. View `MultiInput` demo.
3. Confirm Pills within the input are not selectable.

## Screenshots:
Old:
![Kapture 2019-10-25 at 14 09 27](https://user-images.githubusercontent.com/7979583/67604734-3bc90a00-f731-11e9-9aa6-8dccc0c101af.gif)

New:
![Pill new](https://user-images.githubusercontent.com/7979583/67604738-3ec3fa80-f731-11e9-9fdb-b4fc6dae2e87.gif)
